test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -255,7 +255,7 @@ func (s *snapshottableTestSuite) DefineTests(driver storageframework.TestDriver,
 
 				restoredPod = StartInPodWithVolumeSource(cs, volSrc, restoredPVC.Namespace, "restored-pvc-tester", "sleep 300", config.ClientNodeSelection)
 				cleanupSteps = append(cleanupSteps, func() {
-					StopPod(cs, restoredPod)
+					e2epod.DeletePodWithWait(cs, restoredPod)
```

Comment:
There was a cherrypick conflict here, I updated it to match https://github.com/kubernetes/kubernetes/pull/114217#discussion_r1047266058